__tests__/environment-test.js

```diff
@@ -0,0 +1,125 @@
+"use strict"
+
+jest.dontMock('../environment.js');
+
+describe('environment', () => {
+  it('uses original config attributes if no env key is found', () => {
+    let environment = require('../environment.js').environment;
+
+    let parsedConfig = environment.checkEnvironmentForConfig({
+      key:'fromDefaultConfig'
+    })
+
+    expect(parsedConfig).toEqual({
+      key:'fromDefaultConfig'
+    });
+  })
+
+  it('replaces the config attributes if an env key is found', () => {
+    let environment = require('../environment.js').environment;
+
+    process.env.A_KEY_NAME = 'fromEnv'
+
+    let parsedConfig = environment.checkEnvironmentForConfig({
+      aKeyName:'fromDefaultConfig'
+    })
+
+    expect(parsedConfig).toEqual({
+      aKeyName:'fromEnv'
+    });
+  })
+
+  it('only inserts underscores for every new word', () => {
+    let environment = require('../environment.js').environment;
+
+    process.env.A_KEY_NAME_ID = 'fromEnv'
+
+    let parsedConfig = environment.checkEnvironmentForConfig({
+      aKeyNameID:'fromDefaultConfig'
+    })
+
+    expect(parsedConfig).toEqual({
+      aKeyNameID:'fromEnv'
```

Comment:
I think it would be great if able to test the actual config key, e.g. skipAlreadyAssignedPR, skipAlreadyMentionedPR, delayedUntil